xds/src/main/java/io/grpc/xds/LocalityStore.java

```diff
@@ -73,7 +70,7 @@
 
   void updateDropPercentage(List<DropOverload> dropOverloads);
 
-  void updateOobMetricsReportInterval(long reportIntervalNano);
+  void setLoadStatsStore(@Nullable LoadStatsStore loadStatsStore);
```

Comment:
Inevitably to add this API. The original thought when first implementing `LocalityStore` class, it will always have a `LoadStatsStore` associated with it (same lifecycle). As I mentioned in the PR description, we should not record loads if load reporting is not enabled, so we should allow a `LocalityStore` to have no `LoadStatsStore`. And conceptually, we should allow load reporting to be toggled on/off in the lifetime of a `LocalityStore`.

Reply:
Obsolete.